install.sh

```diff
@@ -49,6 +46,9 @@ while true; do
     '--no-post-install')
         SHOULD_POST_INSTALL=false
         ;;
+    '--no-sudo')
+        NO_SUDO=true
+        ;;
     '--help' | '-h')
         echo "Usage: $0 [--remove,-r] [--dest-dir,-d <installation destination directory (defaults to $DEST_DIR)>] [--prefix-dir,-p <installation prefix directory (defaults to $PREFIX_DIR)>] [--sysconf-dir,-s system configuration destination directory (defaults to $SYSCONF_DIR)] [--no-ectool] [--no-post-install] [--no-pre-uninstall]" 1>&2
```

Comment:
I think the `no-sudo` argument should be documented in the help command

Reply:
also as said in my previous comment, the `no-sudo` argument should also be added to the `pre-uninstall` and `post-install` scripts and be passed accordingly

Reply:
Done. Maybe you could check if it works correctly. In my tests it works, but I could not execute it as a hole since I am on Nixos... :)